.vscode/tasks.json

```diff
@@ -0,0 +1,10 @@
+{
+    "version": "2.0.0",
+    "tasks": [
+        {
+            "label": "buildandinstall",
+            "type": "shell",
+            "command": "cmake --build ${workspaceRoot}/build --config Debug --target install -- -j 26"
```

Comment:
Hardcoding "-j 26" is pretty bad. How about just using Ninja as the build tool, and letting it worry the core count?

Reply:
Removed that